@navikt/core/react/src/form/combobox/Input/Input.tsx

```diff
@@ -132,6 +132,10 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
           if (activeDecendantId || value) {
             e.preventDefault();
           }
+        } else if (e.key === "Escape") {
+          if (isListOpen) {
+            e.preventDefault(); // Prevents the Escape on keydown from closing en encasing Modal, as Combobox reacts on keyup.
```

Comment:
```suggestion
            e.preventDefault(); // Prevents closing an encasing Modal, as Combobox reacts on keyup.
```
Typo + simplification since we already can see from the code that it has to do with Escape keydown.